src/components/app-components/Navbar.tsx

```diff
@@ -49,20 +48,24 @@ class Navbar extends React.Component {
 
                     <Box className={'navbar-items'} id={'navbar-right'}>
                         <Show show={userIsLoggedIn !== true}>
-                            <a href={'/login'}>
+                            <a href={'/login'} id={'login-button'}>
                                 Log In
                             </a>
 
-                            <a href={'/signup'}>
+                            <a href={'/signup'} id={'signup-button'}>
                                 <Button type={'secondary'} style={{fontWeight: 500}}>
                                     Sign Up
                                 </Button>
                             </a>
                         </Show>
 
 
-                        <Show show={userIsLoggedIn}>
-                            <Button type={'secondary'} style={{fontWeight: 500}} onClick={() => {
+                        <Show show={userIsLoggedIn === true}>
+                            <a href={'/dashboard'} id={'dashboard-button'} style={{marginRight: '15px'}}>
```

Comment:
There's a bug going on here where react is displaying "dashboard", but using the href for the login and I don't understand whats happening. I think the content inside these two Show components are too similar and it's getting them confused. We'll see if this keeps happening and I might change this to a function or something that could calculate where it needs to go instead of hardcoding it like this.